README.md

```diff
@@ -1,24 +1,79 @@
+[Clean Code](https://www.sonarsource.com/solutions/clean-code/?utm_medium=referral&utm_source=github&utm_campaign=clean-code&utm_content=sonarqube-scan-action).
 
+[SonarCloud](https://www.sonarsource.com/products/sonarcloud/) is a widely used static analysis solution for continuous code quality and security inspection.
+It helps developers identify and fix issues in their code that could lead to bugs, vulnerabilities, or decreased development velocity.
+SonarCloud supports the most popular programming languages, including Java, JavaScript, TypeScript, C#, Python, C, C++, and [many more](https://www.sonarsource.com/knowledge/languages/).
 
-## Quickstart
+## Requirements
 
-1. Create your account on SonarCloud. Sign up for free now if it's not already the case! [SonarCloud Sign up](https://www.sonarsource.com/products/sonarcloud/signup/).
-2. Create a new manual organization and manual project on SonarCloud.
-3. Follow the in-product tutorial to set up the analysis with this Action.
-4. When your workflow runs, the **SonarCloud Scan** will execute and send analysis results to [SonarCloud](https://sonarcloud.io/).
+* Create your account on SonarCloud. Sign up for free now if it's not already the case! [SonarCloud Sign up](https://www.sonarsource.com/products/sonarcloud/signup/?utm_medium=referral&utm_source=github&utm_campaign=sc-signup&utm_content=signup-sonarcloud-listing-x-x&utm_term=ww-psp-x)
+* The repository to analyze is set up on SonarCloud. [Set it up](https://sonarcloud.io/projects/create) in just one click.
 
+## Usage
 
-## Secrets
+Project metadata, including the location of the sources to be analyzed, must be declared in the file `sonar-project.properties` in the base directory:
 
-The analysis requires the token `SONAR_TOKEN` to be available. You can set it up as a new secret directly within CodeCatalyst and generate it in your SonarCloud security settings [here](https://sonarcloud.io/account/security/).
+```properties
+sonar.organization=<replace with your SonarCloud organization key>
+sonar.projectKey=<replace with the key generated when setting up the project on SonarCloud>
+
+# This is the name and version displayed in the SonarCloud UI.
+#sonar.projectName=windows-msbuild-gh-actions-sq
+#sonar.projectVersion=1.0
+sonar.exclusions=venv/**
+
+# relative paths to source directories. More details and properties are described
+# in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
+sonar.sources=.
+
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
+```
+The workflow, usually declared in `.codecatalyst/workflows/build.yaml`, looks like:
+
+```yaml
+Name: SonarCloudAnalysis
+SchemaVersion: "1.0"
+
+# Optional - Set automatic triggers.
+Triggers:
+  - Type: Push
+# Required - Define action configurations.
+Actions:
+  SonarCloudScanAction:
+    # Identifies the action. Do not modify this value.
+    Identifier: sonar/sonarcloud-scan@v1.0.0
```

Comment:
For now, the name of the action is `sonarr66kw`, which is not ideal (see [the action here](https://codecatalyst.aws/spaces/Sonar/projects/SonarCloudScanAction/actions/sonar%2Fsonarr66kw/view)). 
I tried renaming it, but I don't have permission.
Once we publish the action we will not be able to change the name.